src/style.css

```diff
@@ -2,0 +1,97 @@
+@font-face {
+  font-family: Avenir;
+  src: local('Avenir');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: local('Avenir-Book');
```

Comment:
Ensure that the local font 'Avenir-Book' is available on the user's system. If not, consider providing a fallback web font.